application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -54,20 +62,18 @@ private boolean messageHasNoMediaAttached(Message message) {
                 && !message.getContentRaw().contains("http");
     }
 
-    private RestAction<Message> dmUser(Message message) {
+    private MessageCreateData warningMessage(Message message) {
```

Comment:
its more a notification than a warning. also, since its a method, the name should have a verb, such as `create...`.

Reply:
ok I will do it